tests/rpc/test_rpc_apiserver.py

```diff
@@ -431,14 +431,14 @@ def test_api_profit(botclient, mocker, ticker, fee, markets, limit_buy_order, li
                        'latest_trade_date': 'just now',
                        'latest_trade_timestamp': ANY,
                        'profit_all_coin': 6.217e-05,
-                       'profit_all_fiat': 0,
+                       'profit_all_fiat': ANY,
```

Comment:
I think instead of using ANY i would use the real outcome (which is `0.76748865` in both instances).

It's mocked in conftest, and price is set to 12345.0.